Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -116,8 +116,14 @@ export async function run(artifactToolPath: string): Promise<void> {
                 projectId,
                 feedId,
                 packageName);
+            
+            if(highestVersion != null) {
+                version = artifactToolUtilities.getVersionUtility(tl.getInput("versionPublishSelector"), highestVersion);
+            }            
 
-            version = artifactToolUtilities.getVersionUtility(tl.getInput("versionPublishSelector"), highestVersion);
+            if(version == null) {
```

Comment:
should this be an else to `highestVersion != null`? Or maybe moved to the next outer scope?

Reply:
no, i don't think so?  highestVersion check is on line 120, and this one is to check if version ended getting a usable value for downstream publish.  If it (version) didn't, the publish will fail.